docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -108,15 +124,21 @@ The classic (original) way to use roles is with the ``roles`` option for a given
         - common
         - webservers
 
-When you use the ``roles`` option at the play level, for each role 'x':
+When you use the ``roles`` option at the play level, each role 'x' looks for a ``main.yml/main.yaml/main`` in the following directories:
```

Comment:
I find this formatting confusing: ``main.yml/main.yaml/main``.   How about:
looks for a ``main.yml`` (also`` main.yaml`` and ``main``)

That's how it's described at the start of the roles directory structure section.